packages/datagateway-common/src/detailsPanels/isis/instrumentDetailsPanel.component.tsx

```diff
@@ -46,8 +90,8 @@ const InstrumentDetailsPanel = (
         textColor="secondary"
         indicatorColor="secondary"
         scrollButtons="auto"
-        value={value}
-        onChange={(event, newValue) => setValue(newValue)}
+        value={selectedTab ?? DEFAULT_TAB}
```

Comment:
I agree with having the `useEffect` on line 77 but I'm worry of a slight disconnect if we have 
```
selectedTab ?? DEFAULT_TAB
```
here. Just on the off chance (I'm sure it's unlikely) that the `value` here is incorrect because it's out of sync with `selectedTab`? Idk, maybe it doesn't matter but I feel that if we just have 
```
value={selectedTab}
```
here, then we're always in sync with what's in the store and we haven't got to worry. Feel free to dissuade me on this!

Reply:
Unfortunately MUI doesn't allow `undefined` value. `selectedTab` can only be `undefined` when the details panel is first rendered (and has *never* been rendered before). Upon first render the store will be synced with the selected tab (always `DEFAULT_TAB` for initial render).

Reply:
Oh of course, I knew there must be a little more to it than that. Thanks for letting me know :)